app-laptop/msi-keyboard/msi-keyboard-1.0.ebuild

```diff
@@ -0,0 +1,25 @@
+# Copyright 1999-2019 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=7
+
+inherit udev
+
+DESCRIPTION="Control backlight of MSI laptop keyboards"
+HOMEPAGE="https://github.com/makkarpov/msi-keyboard"
+SRC_URI="https://github.com/makkarpov/msi-keyboard/archive/v${PV}.tar.gz -> ${P}.tar.gz"
+
+LICENSE="GPL-3"
+SLOT="0"
+KEYWORDS="~amd64"
+
+RDEPEND="dev-libs/hidapi"
+PATCHES=(
+		"${FILESDIR}"/${P}-cflags.patch
```

Comment:
One extra tab.

Reply:
I think it's good, i have removed extra tab. Say me if it's bad. I use KDE with kate to edit my ebuild, i don't know what is the good tab size for gentoo ebuild

Reply:
I have removed old patch and add new patch with cflag and ldflag